### ISSUE TITLE
In the Windows 10 environment, calling the local model qwen2.5-code: 7b will not create project files

### ISSUE THREAD

**Issue by flyskyz** [coleam00/bolt.new-any-llm#292]
### Describe the bug

In the Windows 10 environment, calling the local model qwen2.5-code: 7b will not create project files
![image](https://github.com/user-attachments/assets/ff12a6c0-f0ac-4d14-a63a-a8c641de9657)

### Link to the Bolt URL that caused the error

https://github.com/user-attachments/assets/ff12a6c0-f0ac-4d14-a63a-a8c641de9657

### Steps to reproduce

1. Ask questions
2. Can reply to results normally
3. Cannot create project files
4. After repairing, it still cannot be created

### Expected behavior

Can create project files normally

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]
window10

**Comment by tautist** [comment]
Go to your bolt directory delete node_modules folder with all contents and package-lock.json.
In terminal navigate to main directory run: npm install --verbose

I'm on win 10 and there seems to be an issue with remix-run. I have it writing code now but working on getting the preview to work

**Comment by titan2665** [comment]
> Go to your bolt directory delete node_modules folder with all contents and package-lock.json. In terminal navigate to main directory run: npm install --verbose
> 
> I'm on win 10 and there seems to be an issue with remix-run. I have it writing code now but working on getting the preview to work

Hi, i had the same issue, and also my chats werent saved, so i deleted the files you mentionned. Now my bolt can't detect my already installed models and ollama list command won't work too. D you have any idea? Thanks

**Comment by tautist** [comment]
Did you create the folder and file in the C:\BOLT.NEW-ANY-LLM and add the relevant info? e.g. 
FROM qwen2.5-coder:7b
PARAMETER num_ctx 32768

the step is halfway down the git

**Comment by titan2665** [comment]
> Did you create the folder and file in the C:\BOLT.NEW-ANY-LLM and add the relevant info? e.g. FROM qwen2.5-coder:7b PARAMETER num_ctx 32768
> 
> the step is halfway down the git

Yes i did, also i reinstalled rn so it works again but i still have the issue of chats not saved and preview not running.

**Comment by tautist** [comment]
I've noticed with messing around 7b is hit or miss for display and 14b and 32b almost always code and display previews

**Comment by titan2665** [comment]
I'm actually running the light version : 0.5b, you think it could be a reason for these issues?
 Also i don't understand why chats are not saved.

**Comment by tautist** [comment]
i have not tried the 0.5b, but i'm going to assume its an issue with your VRAM?. I don't think the chats and projects are stored locally unless you download the code from the window.

someone better versed might be able to chime in, right now with the 14b for me, its 18.3GB in my memory 50%cpu load and rarely spilling over to my GPU. 

if i sit with bolt for an entire project and download its fine, if i shut down and come back to it the chats are saved but the model forgets where it is. i would suggest trying a larger model

**Comment by titan2665** [comment]
thank you very much for your replies, i'll try to do so and give a feedback !

**Comment by loictr** [comment]
The model does not seam to respond with the asked format with the <boltAction> tags like the examples in the prompt: 
https://github.com/coleam00/bolt.new-any-llm/blob/main/app/lib/.server/llm/prompts.ts

instead it replies with things like:

> 3. **Create `app.py`**
> 
>    ```python
>    from flask import Flask, request, jsonify
